notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
<p>Not - You have a few extra spaces between some words - like in step 3 I see an extra space after "Install" and anywhere else you see a break in the green highlighted text. That's a good and quick way to check for any additional spaces in the text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I can't find extra spaces on my end :D How do you open the notebook? I double checked on my VSCode that shows spaces they were all single.

Reply:
This formatting issue might be caused by NBReview.

Reply:
If you don't see it in the notebook outside of NBReview, it should be fine.